mlx/ops.h

```diff
@@ -1046,6 +1046,43 @@ array conv2d(
     int groups = 1,
     StreamOrDevice s = {});
 
+/** General convolution with a filter */
+array convNd(
+    const array& input,
+    const array& weight,
+    std::vector<int> stride = {},
+    std::vector<int> padding_lo = {},
+    std::vector<int> padding_hi = {},
```

Comment:
Just curious: why is  there a `padding_lo` and `padding_hi` in C++? Also is that padding for the left and right sides separately? It is not exposed to python?

Reply:
Yes, that is padding left and right separately - and good point on the not exposed to python - I'll add it there